src/lxc/cmd/lxc_usernsexec.c

```diff
@@ -53,6 +53,8 @@
 #define MS_SLAVE (1 << 19)
 #endif
 
+extern int lxc_log_fd;
```

Comment:
I fail to see how this is supposed to work so @hallyn I leave this to you.

Reply:
D'oh!

Reply:
I mean, I'm not completely sure if a) overwriting `extern int lxc_log_fd` is save b) if this actually logs something when it is set to `STDERR_FILENO`.

Reply:
Can you elaborate?  It's impolite, but lxc_log_fd is not defined static, so lxc_usernsexec.c should be able to fetch it.

Reply:
I can confirm that it does log stuff :). Is there some reason it wouldn't be safe? Nothing else is initializing it.

Reply:
I could add a lxc_set_log_fd() or something if you want.

Reply:
lxc_set_log_fd() sounds worthwhile, or perhaps it should be handled through lxc_log_init().

But for now I think this is fine.

Reply:
Oh I'm not objecting. I just don't feel comfortable acking this by merging it. :)

Reply:
Ok, thanks for clarifying.  IIRC there is someone looking at a logging overhaul, but if not, maybe I'll consider pursuing that for lxc 4.0.